Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2098,22 +2099,41 @@ void CGContextShowGlyphsWithAdvances(CGContextRef context, const CGGlyph* glyphs
 #pragma endregion
 
 #pragma region Drawing Operations - Basic Shapes
+
+HRESULT __CGContext::ClearRect(CGRect rect) {
+    PushBeginDraw();
+    auto endDraw = wil::ScopeExit([this]() { PopEndDraw(); });
+
+    auto factory = Factory();
```

Comment:
nit: shouldn't use auto for this since type isn't specified on right/unrepresentable